sdk/appconfiguration/Azure.Data.AppConfiguration/src/ConfigurationSetting.cs

```diff
@@ -16,8 +16,10 @@ public sealed class ConfigurationSetting : IEquatable<ConfigurationSetting>
     {
         private IDictionary<string, string> _tags;
 
-        // TODO (pri 3): this is just for deserialization. We can remove after we move to JsonDocument
-        internal ConfigurationSetting() { }
+        internal ConfigurationSetting()
+        {
+            ETag = new ETag(default);
```

Comment:
ETag is a struct, it is initialized with a value equal to `new ETag(default)` by default.